src/components/PageSwitcher/PageSwitcher.js

```diff
@@ -35,18 +35,29 @@ const predict = translations.types.predict;
 
 export function PageSwitcher() {
 
+  // For auxEnable settings
```

Comment:
logic to control aux at very high level since it needs to happen before connect and only show toggle in certain modules